zerver/lib/email_mirror.py

```diff
@@ -161,12 +162,23 @@ class ZulipEmailForwardUserError(ZulipEmailForwardError):
     pass
 
 def send_zulip(sender: str, stream: Stream, topic: str, content: str) -> None:
+    trunc_topic = truncate_topic(topic)
+    if len(trunc_topic) < len(topic):
+        pdlte = datetime.datetime.today()
+        pdgt = pdlte - datetime.timedelta(days = 7)
+        if not (Message.objects.filter(
+                subject = trunc_topic,
+                recipient = get_stream_recipient(stream.id),
+                date_sent__lte = pdlte,
+                date_sent__gt = pdgt).exists()):
```

Comment:
Wouldn't it be more reable to just use ``date_sent__gt = timezone_now() - timedelta(days=7)``, and remove the ``date_sent__lte`` part (is it needed for anything?)?